ParkingLot/ParkingLot.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ParkingLotManagement
+{
+    public class ParkingLot
+    {
+        private readonly int capacity = 10;
+        private string name;
+
+        private Dictionary<Ticket, string> ticket2Car = new Dictionary<Ticket, string>();
+        public ParkingLot(string name)
+        {
+            Name = name;
+        }
+
+        public ParkingLot()
+        {
+            Name = "Defalt_Name";
+        }
+
+        public int Capacity { get => capacity; }
+        public Dictionary<Ticket, string> Ticket2Car { get => ticket2Car; set => ticket2Car = value; }
+        public string Name { get => name; set => name = value; }
+
+        public string Fetch(Ticket ticket)
+        {
+            if (ticket == null || !Ticket2Car.ContainsKey(ticket) || ticket.IsValid == false)
+            {
+                throw new WrongTicketExceptoion("Unrecognized parking ticket.");
+            }
+
+            ticket.IsValid = false;
```

Comment:
ticket.IsValid is not needed if delete cars from tick2Car dictionary after fetch and use the dictionary to check if ticket's corresponding car exists.